source/module_io/read_wfc_lcao.cpp

```diff
@@ -0,0 +1,320 @@
+/**
+ * @file read_wfc_lcao.cpp
+ * @author kirk0830
+ * @brief this file is part of the project of removal of class Local_Orbital_Wfc (LOWF)
+ * @version 0.1
+ * @date 2024-06-07
+ * 
+ * @copyright Copyright (c) 2024
+ * 
+ */
+#include "module_io/read_wfc_lcao.h"
+#include "module_base/formatter.h"
+#include "module_base/tool_quit.h"
+#include <fstream>
+#include <regex>
+#include <cassert>
+
+template<typename T>
+void ModuleIO::read_abacus_lowf(const std::string& flowf, 
```

Comment:
Where is the function "read_abacus_lowf" used?

Reply:
It seems the "abacus" name is redundant.

Reply:
> Where is the function "read_abacus_lowf" used?

This name is designed to keep accordance with the function read numerical atomic orbital file `read_abacus_orb`. This function is for taking place of `LOWF::allocate_k`, presently there is no place where this function is called. Next I will insert a procedure in ESolver_KS_LCAO to do something like wavefunction initialization.

Reply:
> Where is the function "read_abacus_lowf" used?

This name is designed to keep accordance with the function read numerical atomic orbital file `read_abacus_orb`. This function is for taking place of `LOWF::allocate_k`, presently there is no place where this function is called. Next I will insert a procedure in ESolver_KS_LCAO to do something like wavefunction initialization.